frontend/test/__support__/ui.tsx

```diff
@@ -71,6 +72,53 @@ export function renderWithProviders(
     ...options
   }: RenderWithProvidersOptions = {},
 ) {
+  const { wrapper, store, history } = getTestStoreAndWrapper({
+    mode,
+    initialRoute,
+    storeInitialState,
+    withRouter,
+    withKBar,
+    withDND,
+    withUndos,
+    customReducers,
+    theme,
+    ...options,
+  });
+
+  const utils = testingLibraryRender(ui, {
+    wrapper,
+    ...options,
+  });
+
+  return {
+    ...utils,
+    store,
+    history,
+  };
+}
+
+export function renderHookWithProviders<TProps, TResult>(
+  hook: (props: TProps) => TResult,
+  options?: Omit<RenderHookOptions<TProps>, "wrapper"> &
+    RenderWithProvidersOptions,
+) {
+  const { wrapper, store } = getTestStoreAndWrapper(options);
+  const renderHookReturn = renderHook(hook, { wrapper, ...options });
+
+  return { ...renderHookReturn, store };
+}
+
+export function getTestStoreAndWrapper({
+  mode = "default",
```

Comment:
since the default value has been moved to `getTestStoreAndWrapper` is it better to remove the default value from `renderWithProviders`? I'd expect both `renderWithProviders` and `renderHookWithProviders` to look similar, and don't provide default value for the hook version.